src/ast.cr

```diff
@@ -5,9 +5,9 @@ module Mint
 
     getter unified_modules, unified_locales
 
-    def initialize(@type_definitions = [] of TypeDefinition,
-                   @operators = [] of Tuple(Int64, Int64),
-                   @keywords = [] of Tuple(Int64, Int64),
+    def initialize(@operators = [] of Tuple(Parser::Location, Parser::Location),
+                   @keywords = [] of Tuple(Parser::Location, Parser::Location),
```

Comment:
Instead of a `Tuple` I'd suggest using a `record`, or at least a `NamedTuple` for readability purposes.
Otherwise IMHO it's a bit hard to understand what the values are representing.